### PR TITLE
Update Cmdkey

### PR DESCRIPTION
**Pull request by frack113**
Cmdkey option can be shorted to only 1 caracter

image

But `cmdkey /a:server03 /usersssssssss:mikedan /p:Kleo` works too 😟 

cc: @nasbench

### CODE REVIEW

**Comment by nasbench** [comment]
LOL....This utility is insane, they just allow everything as long as it starts with the correct letter 😢

File: rules/windows/process_creation/proc_creation_win_mstsc.yml [hunk 1]
```diff
@@ -21,9 +22,9 @@ detection:
         - OriginalFileName: 'cmdkey.exe'
     selection_cmdkey_cli:
         CommandLine|contains|all:
-            - '/generic:'
-            - '/user:'
-            - '/pass:'
+            - '/g'
+            - '/u'
+            - '/p'
```
Comment (Neo23x0) [inline rules/windows/process_creation/proc_creation_win_mstsc.yml:27]: Can't we put a space character before the slashes? 
Every character counts

Comment (frack113) [reply rules/windows/process_creation/proc_creation_win_mstsc.yml:27]: Yes We Can 🎶

File: rules/windows/process_creation/proc_creation_win_local_system_owner_account_discovery.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ detection:
         - '\quser.exe'
         - '\qwinsta.exe'
     - Image|endswith: '\cmdkey.exe'
-      CommandLine|contains: '/list'
+      CommandLine|contains: '/l'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_local_system_owner_account_discovery.yml:24]: This one as well

Comment (Neo23x0) [reply rules/windows/process_creation/proc_creation_win_local_system_owner_account_discovery.yml]: Is everyone working on Sundays? 🙌😅

**Pull request merged**